modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/parser/java/JavaConnectionProviderModelParser.java

```diff
@@ -95,6 +97,8 @@ public JavaConnectionProviderModelParser(JavaExtensionModelParser extensionModel
         .orElse(ExtensionModel.class.getClassLoader());
 
     collectAdditionalModelProperties();
+    // TODO W-10621050
+    this.minMuleVersion = calculateConnectionProviderMinMuleVersion(element);
```

Comment:
You can already remove this code and uncomment the new lines that have the W-12398760 mentioned

Reply:
Done

Reply:
You probably will need to update the serialized extension models after this.